### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Alabi12**
### **WHAT?**
1. Contact form was added to the portfolio
2.  Both the index.html and css files were updated to have the contact form created with the earlier portfolio completed version.

### **WHY?**
1. The contact me form is a necessary element of a portfolio to let a user contact the maker.
2. Also, this completed version of the portfolio showcases my skills and abilities regarding coding and designing a portfolio website-mobile version.
3. The files are checked for linter errors, correct GitHub flow and proper documentation of the repository.

### **HOW?**
1. The files can be tested for linter errors using GitHub Actions, Lighthouse, Webhint and Stylelint.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -171,7 +171,21 @@ <h3>CSS</h3>
...
+    <form action="https://formspree.io/f/mnqrdydw" method="post">
+      <label for="name"></label>
+      <input name="name" placeholder="Your Name" id="name" type="name" required size="30">
+      <label for="email"></label>
+      <input name="Email" placeholder="Your Email" id="email" type="email" required>
+      <textarea name="textarea" placeholder="Write your message here" id="textarea" cols="30" rows="10" required aria-setsize="500"></textarea>
+    <button class="form-btn" type="submit">Get It Touch</button>
+    </form>
+    </div>
+  </section>
```
Comment (mckent05) [inline index.html]: - Currently, I cannot see the styling you have applied to your contact form in this project. Please try to fix this 🙏 

Your project currently looks like this :
last project contact

Figma Template-
figma 1contact

- Also, please remember to implement button interactions such as hover, active, etc. 👍 👍 

Figma (Button interactions) -
figma 1button

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi guys, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Descriptive PR title and summary 💯 💯 
- Linters are passing ✔️ ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css [hunk 2]
```diff
@@ -406,4 +406,4 @@ form > ul {
   padding: 12px 16px;
   background: #fff;
   border-radius: 8px;
-}
\ No newline at end of file
+}
```
Comment (ichala) [inline portfolio.css:409]: - [ ] as requested by the previous reviewer  kindly try to implement button interactions such as hover, active .; etc as shown in Figma design

192822959-9f0ccbd4-4720-4afd-a81c-7f293746ffbf

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @Alabi12  , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design ✅ 
- Form submission works ✅ 

 ## Required Changes ♻️

- [ ] Check the comments below

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @Alabi12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Working form✔️
Design looks great✔️
No linter errors✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**